v3.9/introduction/index.md

```diff
@@ -43,7 +43,7 @@ custom_css: css/intro.css
   </div>
   <div class="col-md-6">
     <h3 style="margin-top: 5px">Performance</h3>
-    <p>{{site.prodname}} uses the Linux kernel’s built-in highly optimized forwarding and access control capabilities to deliver native Linux networking dataplane performance, typically without requiring any of the encap/decap overheads associated with first generation SDN networks. {{site.prodname}}’s control plane and policy engine has been fine tuned over many years of production use to minimize overall CPU usage and occupancy.</p>
+    <p>{{site.prodname}} uses the Linux kernel’s built-in highly optimized forwarding and access control capabilities to deliver native Linux networking dataplane performance, typically without requiring any of the encap/decap overheads associated with first generation SDN networks. {{site.prodname}}’s control plane and policy engine has been fine tuned over many years of production use to <a href="https://itnext.io/benchmark-results-of-kubernetes-network-plugins-cni-over-10gbit-s-network-36475925a560">minimize overall CPU usage and occupancy</a></p>
```

Comment:
I don't think we should link to this report.

Firstly, it's out of date and there is an updated report. But also, while it does seem a fair comparison, performance testing is hard, there are a lot of similar reports on performance, and I'd rather not "endorse" one report or another in this way.

Reply:
Ok fair enough. I’ll remove that part :).

Reply:
ok,... i deleted the `a href` and alluded to what you said above.  that way the statement isnt entirely anecdotal.. :) let me know if that works better (if not ill just delete this sentence change i think)